### PR TITLE
2.1 AAP-4122 Added allowlisted domains (#414)

### PR DESCRIPTION
**Pull request by ianf77**
* Added allowlisted domains

Added list of allowlisted domains and other requirements

Add in the doc from config AAP the domains which need to be whitelisted

https://issues.redhat.com/browse/AAP-4122

* Added allowlisted domains

Corrections and addressed comments

Add in the doc from config AAP the domains which need to be whitelisted

https://issues.redhat.com/browse/AAP-4122

Title affected:  ./hub/configuring-private-hub-rh-certified ([section 1.2](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html/managing_red_hat_certified_and_ansible_galaxy_collections_in_automation_hub/assembly-synclists#proc-create-synclist))

### CODE REVIEW

File: downstream/modules/automation-hub/proc-create-synclist.adoc [hunk 1]
```diff
@@ -12,11 +12,22 @@ By default, all Red Hat Certified collections are included in your initial organ
 
 * You have a valid Ansible Automation Platform subscription.
 * You have Organization Administrator permissions for cloud.redhat.com.
+* Ensure that the following domain names are part of either the firewall or the proxy's allowlist for successful connection and download of collections from {HubName} or Galaxy server:
```
Comment (ariordan-redhat) [inline downstream/modules/automation-hub/proc-create-synclist.adoc:15]: {HubName} and {HubNameStart} aren't rendering properly in my local docs build. I think we need to create a PR to main to fix this.

Try adding the following line to downstream/titles/hub/configuring-private-hub-rh-certified/master.adoc

```
include::attributes/attributes.adoc[]
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
The Hub attributes aren't rendering properly: need a PR to `main` to fix this.
Rebase the branch for this backport PR against `main` once the fix is merged.

File: downstream/modules/automation-hub/proc-create-synclist.adoc (hunk 1 above)
Comment (ianf77) [reply downstream/modules/automation-hub/proc-create-synclist.adoc:15]: In main?

Comment (ariordan-redhat) [reply downstream/modules/automation-hub/proc-create-synclist.adoc:15]: Yes. I've just created a PR: #444

Comment (ariordan-redhat) [reply downstream/modules/automation-hub/proc-create-synclist.adoc:15]: Once that's merged, you can rebase the backport PR branch against upstream/main and then update this PR.

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Comment by ariordan-redhat** [comment]
This PR includes PR #444

**Pull request merged**